web/src/components/merchant/sign-in/hooks/useSignInForm.tsx

```diff
@@ -34,21 +40,37 @@ const useSignInForm = () => {
   const {formState, handleSubmit, register} = useForm<SignInData>({
     mode: 'onChange',
   });
-  const [generalError, setGeneralError] = useState();
+  const [generalError, setGeneralError] = useState<Error | undefined>();
+  const history = useHistory();
 
   const validations = {
     email: register({
-      required: Errors.EMAIL_EMPTY,
+      required: true,
     }),
     password: register({
-      required: Errors.PASSWORD_EMPTY,
+      required: true,
     }),
   };
   const disabled = !formState.isValid;
 
   const onSubmit = handleSubmit(async (values: SignInData) => {
-    setGeneralError(null);
-    // TODO: Handle user sign in
+    setGeneralError(undefined);
```

Comment:
Actually did you do `setGeneralError(undefined)` because you want to "reset" the general error message? I'm thinking maybe can leave a comment stating this!

Reply:
Yeah! It's to reset the error. Hahaha I thought it was clear enough. But okay will add that comment.